modules/csharp/pages/working-with-vector-search.adoc

```diff
@@ -133,16 +134,14 @@ include::csharp:example$code_snippets/VectorSearch.cs[tags=vs-create-index]
 
 ----
 
-. First, create the standard configuration, setting up an expressions, number of dimensions and number of centroids for the vector embedding.
-
-. Next, get the colors collection from the database.
+. First, create the standard configuration, setting up an expression, number of dimensions and number of centroids for the vector embedding.
 
-. Finally, create a vector index, colors_index, on the collection with the configuration.
+. Next, create a vector index, `colors_index`, on a collection and pass it our configuration.
 
 === Create Vector Index Embeddings from a Predictive Model
 
 This method generates vectors to be indexed for each document at the index time by using the `prediction()` function.
-The key difference to note is that the `config` object uses the output of the `prediction()` function as the `expression` parameter to generate the color embedding.
+The key difference to note is that the `config` object uses the output of the `prediction()` function as the `expression` parameter to generate the vector index.
```

Comment:
This section has a snippet following it that doesn't really tell the whole story.  The user needs some guidance on the way that prediction function itself interacts with the color model (it's similar, if not identical, to predictive query).  That is, that the `colorInput` comes from a document property into the prediction function, and `vector` is a key on the result dictionary that is written by the prediction function and that is why the syntax `{\"colorInput\": color}).vector` is used.  The generic naming of `vector` could mislead people into thinking it is just a magic name.

Reply:
Addressed